src/lib/components/table/evm-transactions/EvmTransactionsTable.tsx

```diff
@@ -39,6 +39,7 @@ export const EvmTransactionsTable = ({
     "48px",
     "minmax(180px, 2fr)",
     "minmax(250px, 1fr)",
+    "minmax(247px, 1fr)",
```

Comment:
need to add a new prop to this component, since block evm txs don't need to show the timestamp
```
showTimestamp? = false
```